src/index.js

```diff
@@ -23,52 +28,143 @@ const loadManifest = (manifest, path) => (
   })
 );
 
-const changeVersion = async () => {
-  const display = console.log; // eslint-disable-line no-console
+function versionPackage(versionText) {
```

Comment:
I'm not sure how clear that function name sounds. I suggest making it `setPackageVersion` instead. Same comment for other similar function names.